charts/kubeai/values.yaml

```diff
@@ -44,6 +44,22 @@ resourceProfiles:
       nvidia.com/gpu: "1"
       cpu: "6"
       memory: "24Gi"
+    nodeSelector:
+      cloud.google.com/gke-accelerator: "nvidia-l4"
+    tolerations:
+      - key: "nvidia.com/gpu"
+        operator: "Equal"
+        value: "present"
+        effect: "NoSchedule"
+    affinity:
```

Comment:
Agreed - we can keep the tolerations though

Reply:
Ah yeah true. Let's keep the toleration.

Reply:
I think we can accomplish a resource profile that works across most popular cluster types by leveraging how affinity `nodeSelectorTerms` are OR'd together instead of AND'd. Will open a new issue to track this. We should still remove the affinity & node selector in here for now.